.github/workflows/test.yaml

```diff
@@ -0,0 +1,19 @@
+name: web
```

Comment:
This file is for running `flutter test` in github actions.
Configuration is so simple 😄